src/conv/k-g/k_parser.cpp

```diff
@@ -189,6 +190,9 @@ bool parse_k
 				partLinesRead = 0;
 				partTitle     = "";
 			    }
+			    else if ((command.size() == 3)) {

```

Comment:
Test the whole string: `else if ((command.size() == 3) && (command[1] == "ADAPTIVE") && (command[2] == "FAILURE")) {`